### PR TITLE
Feature/287/liking a streetcode

### PR DESCRIPTION
**Pull request by Shavronskyy**
## Checklist before requesting a review
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Have you lint your code locally before submission?
- [x] If that PR have tests
- [x] I have performed a self-review of my code

### CODE REVIEW

**Review by PavloIeremeiew — APPROVED** [review]

File: Streetcode/Streetcode.BLL/DTO/Users/UserDTO.cs [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+namespace Streetcode.BLL.DTO.Users
 {
     public class UserDTO : UserRegisterDTO
     {
         public bool PhoneConfirmed { get; set; }
 
         public bool EmailConfirmed { get; set; }
 
-        public bool TwoFactorEnabled { get; set; }        
+        public bool TwoFactorEnabled { get; set; }
```
Comment (metrondir) [inline Streetcode/Streetcode.BLL/DTO/Users/UserDTO.cs:11]: we use that? i think we need to erase that

File: Streetcode/Streetcode.WebApi/Controllers/Likes/LikeController.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Streetcode.BLL.DTO.Likes;
+using Streetcode.BLL.MediatR.Likes.PushLike;
+    
+namespace Streetcode.WebApi.Controllers.Likes
+{
+    public class LikeController : BaseApiController
+    {
```
Comment (metrondir) [inline Streetcode/Streetcode.WebApi/Controllers/Likes/LikeController.cs:8]: Add Authorize for like

**Review by metrondir — APPROVED** [review]

File: Streetcode/Streetcode.BLL/DTO/Users/UserDTO.cs (hunk 1 above)
Comment (Shavronskyy) [reply Streetcode/Streetcode.BLL/DTO/Users/UserDTO.cs:11]: its not mine

File: Streetcode/Streetcode.WebApi/Controllers/Likes/LikeController.cs (hunk 2 above)
Comment (Shavronskyy) [reply Streetcode/Streetcode.WebApi/Controllers/Likes/LikeController.cs:8]: done
added single authorize for all roles

**Pull request merged**